CHANGELOG.md

```diff
@@ -63,6 +64,7 @@ All notable changes to this project will be documented in this file.
 - Set default value in parent element in _Add Attribute_ modal
 - Values in Dropdown attributes are now sorted alphabetically
 - Values in Dropdown attributes are now searchable
+- Data Importer: Now supports the Boolean attribute with the value 'x'
```

Comment:
I still think `x` is an ambiguous value for booleans in csv. Also it is no rocket science to replace all 'x' with '1' or something else.
But we can add it for now. For the future I'd like to have a configurable list of truthy values.